{{cookiecutter.project_slug}}/.env.example

```diff
@@ -149,5 +149,9 @@ SMTP_VALID_TESTING_DOMAINS='thinknimble.com'
 
 # Testing (NOTE: Heroku and Github Actions will need to have matching values for some of these)
 DJANGO_SUPERUSER_PASSWORD='!!!DJANGO_SECRET_KEY!!!'
-CYPRESS_baseUrl='http://localhost:8080'
 CYPRESS_TEST_USER_PASS='!!!DJANGO_SECRET_KEY!!!'
+{%- if cookiecutter.client_app == "React" %}
+CYPRESS_baseUrl='http://localhost:5173'
```

Comment:
We were using 8089 for local

Reply:
Okay Ty! That change must have occurred after Julep